FrontEnd/src/components/basics/ImageUpload/ImageUpload.module.scss

```diff
@@ -0,0 +1,71 @@
+$border-radius: 0.1875rem;
+.image_input {
+  display: none;
+}
+
+.image_input_label {
+  cursor: pointer;
+}
+
+.delete_wrapper_background {
+  position: absolute;
+  display: none;
+  width: 100%;
+  height: 100%;
+  background: rgba(32, 32, 32, 0.6);
```

Comment:
Should the color be a magic number or a global var? is it already a global var? Will it be reused?

Reply:
yeah, I made it a global var. Good catch!